src/content/platform-integration/macos/platform-views.md

```diff
@@ -118,29 +118,28 @@ class NativeViewFactory: NSObject, FlutterPlatformViewFactory {
 }
 
 class NativeView: NSView {
-  private var _view: NSView
 
   init(
-    frame: CGRect,
     viewIdentifier viewId: Int64,
     arguments args: Any?,
     binaryMessenger messenger: FlutterBinaryMessenger?
   ) {
-    _view = NSView()
-    super.init(frame: frame)
+    super.init(frame: CGRect(x: 0, y: 0, width: 200, height: 200))
+    wantsLayer = true
+    layer?.backgroundColor = NSColor.systemBlue.cgColor
     // macOS views can be created here
-    createNativeView(view: _view)
+    createNativeView(view: self)
   }
-
+    
+    required init?(coder nsCoder: NSCoder) {
+        super.init(coder: nsCoder)
+    }
+    
   func createNativeView(view _view: NSView) {
-    _view.wantsLayer = true
-    _view.layer?.backgroundColor = NSColor.systemBlue.cgColor
-    _view.frame = CGRect(x: 0, y: 0, width: 200, height: 200)
```

Comment:
Why remove the backgroundColor? This is there for consistency across iOS/macOS example code. Similar with the textColor below.

Reply:
@fengjiongmax, please make Chris' suggested changes. thx

Reply:
They were moved to the `init` function, should I move them back here?